### PR TITLE
Update privacy_notice.md

### PR DESCRIPTION
**Pull request by deniseal**
Updates to 2.2.2 to include location sharing

### CODE REVIEW

File: docs/ems/privacy_notice.md [hunk 1]
```diff
@@ -178,10 +179,18 @@ We track usage data for Element hosting services. When you are signed in to your
...
 
 When reporting errors we might collect some information to help us find a solution. This may include your IP address, hostname, full name and e-mail address. This information is collated in the application monitoring platform [Sentry](https://sentry.io/welcome/) and is subject to strict retention policies.
 
+**Location Information**
+
+We may collection location data on you, if you chose to use the static or live location sharing features within the Element app. This includes your longitude, altitude and latitude data in order to accurately calculate your precise location.
```
Comment (Johennes) [inline docs/ems/privacy_notice.md]: We may _collect_?

File: docs/ems/privacy_notice.md [hunk 2]
```diff
@@ -178,10 +179,18 @@ We track usage data for Element hosting services. When you are signed in to your
...
 
 When reporting errors we might collect some information to help us find a solution. This may include your IP address, hostname, full name and e-mail address. This information is collated in the application monitoring platform [Sentry](https://sentry.io/welcome/) and is subject to strict retention policies.
 
+**Location Information**
+
+We may collection location data on you, if you chose to use the static or live location sharing features within the Element app. This includes your longitude, altitude and latitude data in order to accurately calculate your precise location. 
+
+Location data is held within the room in which is shared in, so it will be encrypted in encrypted rooms and not encrypted in rooms where encryption is switched off. You will be shown a disclaimer during your first time using this feature, but please apply caution and consideration when sharing your personal data within the app.
```
Comment (Johennes) [inline docs/ems/privacy_notice.md]: in which _it_ is shared in?

**Review by Johennes — CHANGES_REQUESTED** [review]
I think there are two small grammar issues (though I'm German so I might be wrong 🙈). Otherwise looks good, thank you! 🙂

File: docs/ems/privacy_notice.md (hunk 1 above)
Comment (andybalaam) [reply docs/ems/privacy_notice.md]: If you *choose*?

File: docs/ems/privacy_notice.md (hunk 2 above)
Comment (andybalaam) [reply docs/ems/privacy_notice.md]: in which it is shared? (Remove second "in"?)

File: docs/ems/privacy_notice.md [hunk 3]
```diff
@@ -178,10 +179,18 @@ We track usage data for Element hosting services. When you are signed in to your
...
+
+We use the third-party library [MapLibre](https://maplibre.org/) to provide you with map information.
```
Comment (andybalaam) [inline docs/ems/privacy_notice.md]: I don't think we need to mention MapLibre - this is just some code we use, similar to many other libraries.

Instead, we should mention MapTiler:

The Element clients use the third-party service [MapTiler](maptiler.com) to provide the images used to display maps.

**Review by andybalaam — CHANGES_REQUESTED** [review]

**Comment by bmarty** [comment]
@deniseal do we will have to update our "Data safety" form on the PlayStore/iTunes console?

**Comment by deniseal** [comment]
> @deniseal do we will have to update our "Data safety" form on the PlayStore/iTunes console?

no, @bmarty we've already included location sharing in the latest version of the forms so we should be all covered there. Thanks for checking!

**Pull request merged**